### PR TITLE
Added constant var to all polling sources consume

### PR DESCRIPTION
**Pull request by lcaresia**

### CODE REVIEW

**Review by luancazarine — DISMISSED** [review]

File: platform/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/platform",
-  "version": "1.1.1",
+  "version": "1.1.",
```
Comment (andrewjschuang) [inline platform/package.json:3]:
```suggestion
  "version": "1.2.0",
```

File: platform/lib/constants.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const DEFAULT_POLLING_SOURCE_TIMER_INTERVAL = 6 * 15
```
Comment (andrewjschuang) [inline platform/lib/constants.ts:1]:
```suggestion
export const DEFAULT_POLLING_SOURCE_TIMER_INTERVAL = 60 * 15;

```

File: platform/lib/index.ts [hunk 3]
```diff
@@ -14,6 +14,10 @@ export {
   ConfigurationError,
 } from "./errors";
 
+export {
+  DEFAULT_POLLING_SOURCE_TIMER_INTERVAL
+} from './constants'
```
Comment (andrewjschuang) [inline platform/lib/index.ts:19]:
```suggestion
export {
  DEFAULT_POLLING_SOURCE_TIMER_INTERVAL,
} from "./constants";
```

**Review by andrewjschuang — CHANGES_REQUESTED** [review]
Also run the following and commit the changes:

- `npm i`
- `npm run prebuild`
- `npm run build`

**Review by andrewjschuang — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**